.github/workflows/preDeploy.yml

```diff
@@ -222,3 +222,104 @@ jobs:
             3. Once your PR is deployed to _production_, we start a 7-day timer :alarm_clock:. After it has been on production for 7 days without causing any regressions, then we pay out the Upwork job. :moneybag:
 
             So it might take a while before you're paid for your work, but we typically post multiple new jobs every day, so there's plenty of opportunity. I hope you've had a positive experience contributing to this repo! :blush:
+
+  e2e-tests:
```

Comment:
I'd maybe prefer to just have this run in a separate workflow with the same trigger because this file is kind of crazy already. NAB though I suppose